src/store.rs

```diff
@@ -95,6 +92,41 @@ impl StoreHandle {
 
         Ok(store.into_crypto_store())
     }
+
+    /// Open a crypto store based on IndexedDB, using the given key for
+    /// encryption.
+    ///
+    /// # Arguments
+    ///
+    /// * `store_name` - The name that should be used to open the IndexedDB
+    ///   based database.
+    ///
+    /// * `store_key` - The key that should be used to encrypt the store, for
+    ///   IndexedDB-based stores. Must be a 32-byte array.
+    #[wasm_bindgen(js_name = "openWithKey")]
+    pub async fn open_with_key(
+        store_name: String,
+        mut store_key: Vec<u8>,
+    ) -> Result<StoreHandle, JsError> {
+        use zeroize::Zeroize;
+
+        let mut store_key_slice: [u8; 32] = store_key
```

Comment:
This is not a slice, so maybe should be called `store_key_array`?

Reply:
Indeed. Fixed, thanks.